lib/activerecord/delay_touching.rb

```diff
@@ -87,17 +87,23 @@ def self.touch_records(attr, klass, records)
           changes[column] = current_time
           records.each do |record|
             next if record.destroyed?
```

Comment:
You can remove this line since the `persisted?` method includes this check.
http://apidock.com/rails/ActiveRecord/Persistence/persisted%3F